apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -256,7 +256,10 @@ namespace MWRender
         mRootNode->getOrCreateStateSet()->addUniform(mUniformRainIntensity);
 
         mSky.reset(new SkyManager(sceneRoot, resourceSystem->getSceneManager()));
-        mSky->setCamera(mCamera->getOsgCamera());
+
+        std::vector<osg::Camera*> cameras;
+        mViewer->getCameras(cameras,true);
```

Comment:
Eh, that works too, but mViewer->getCamer<b>a</b>() is easier no? :)

Reply:
I appreciate the patience you have with me :)